torch/_higher_order_ops/torchbind.py

```diff
@@ -106,12 +106,20 @@ def inner(mode, *args, **kwargs):
         return call_torchbind(*args, **kwargs)
 
 
-# TODO: currently we just run the C++ implementation with fake tensors.
-# But we should make it possible to register a fake torchbind implementation.
+# When tracing with fake script object, the call_torchbind op will return a fake tensor
+# When tracing with real script object, the call_torchbind op return a real tensor,
+# we need to convert it to real tensor mannually.
```

Comment:
What's the strategy for tracing with a real script object?
- We run methods with the real script object and FakeTensors? The alternative is running methods with the real script object and real Tensors.
- We run operators with the real script objects and FakeTensors? The alternative is running operators with real script objects and real Tensors

Reply:
> we run methods and operators with real script object and fake tensor inputs 

This is the current status. People implemented meta kernels that handles the real script object + fake tensor inputs.

> The alternative is running methods with the real script object and real Tensors.

Is it possible to get a real tensors while we're tracing a torchbind method/operator? We need to trace it together with real tensor inputs along with fake tensor inputs, right?

Reply:
Thanks for clarifying, can you put down the current approach somewhere?

Reply:
Sure!